### ISSUE TITLE
does not compile (ctypes 0.7.0)

### ISSUE THREAD

**Issue by treinen** [colis-anr/ocaml-libdpkg#2]
After having changed in dune-project the lang from 1.8 to 1.6 (see #1) I obtain

````
File "bindings/types/dpkg_types.ml", line 3, characters 14-25:
Error: Unbound module type Ctypes.TYPE
make: *** [Makefile:4: build] Error 1
````

This is version 0.7.0 of ctypes (I guess we should upgrade the package in debian, right?)

**Comment by Niols** [comment]
Hum, I should try to determine the minimal version of Ctypes that is required. I developed this draft with 0.14.0.

**Comment by Niols** [comment]
Indeed, ctypes ≥ 0.13.0 is required. It is probably possible to compile this lib with earlier versions, but I don't know how.

**Comment by treinen** [comment]
OK. Unfortunately that version of ocaml-ctypes requires ocaml-integer which we hence have to package first for debian before updating the ocaml-ctypes package.

**Comment by Niols** [comment]
Hum, it's a shame. And I have no idea how it was done with previous versions of ctypes. Although apparently, ctypes 0.13.0 is from October 2017.
